src/lib/stitching/mergeSchemas.ts

```diff
@@ -80,7 +80,9 @@ export const incrementalMergeSchemas = (localSchema, version: 1 | 2) => {
   if (version === 1) {
     useStitchingEnvironment(vortexStitchingEnvironmentv1(localSchema))
   } else {
-    useStitchingEnvironment(vortexStitchingEnvironmentv2(localSchema))
+    useStitchingEnvironment(
+      vortexStitchingEnvironmentv2(localSchema, gravitySchema)
```

Comment:
Think it is OK to only integrate the `gravitySchema` for `vortexStitchingEnvironmentv2`? or should we add this implementation to `vortexStitchingEnvironmentv1` as well? Looks like these functions are explicitly versioned.

I remember we chatted about it when we were mini-mobbing but I dont recall the outcome.

Reply:
I'm honestly not sure what v1 is left over from, but relatively sure v2 is what we're consuming with Partner Analytics so seems safe to just use v2 IMO